transformer/cloudfoundrytransformer.go

```diff
@@ -263,21 +299,43 @@ func (t *CloudFoundry) prioritizeAndAddEnvironmentVariables(cfApp collecttypes.C
 			flattenedEnvList := flattenVcapServiceVariables(valueStr)
 			for _, env := range flattenedEnvList {
 				envOrderMap[env.Name] = env
+				vcapEnvMap[env.Name] = []byte(env.Value)
 			}
 		}
+		dbEnv, err := createDbEnvironmentVariable(valueStr, serviceName)
+		if err != nil {
+			logrus.Errorf("Could not create DATABASE_URL environment: %v", err)
+		} else {
+			envOrderMap[dbEnv.Name] = dbEnv
+			vcapEnvMap[dbEnv.Name] = []byte(dbEnv.Value)
+		}
```

Comment:
shouldn't this be done within the if condition above?

Reply:
This comment is moot. I have removed createDbEnvironmentVariable().